CHANGELOG.md

```
@@ -1,5 +1,9 @@
 # sec change log
 
+### v0.3.2
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)